README.md

```diff
@@ -235,6 +235,15 @@ When run with this flag, `publish` will specify updated dependencies in updated
 
 For more information, see the package.json [dependencies](https://docs.npmjs.com/files/package.json#dependencies) documentation.
 
+May also be configured in `lerna.json`:
+
+```js
+{
+  ...
+  "exact": true
+}
+```
+
```

Comment:
We already describe the `lerna.json` "durable" config pattern elsewhere, I think this is implied (much like #598, which only made a single line change and no docs updated).